cva6/docs/VerifPlans/csr_access/VP_IP003.yml

```diff
@@ -0,0 +1,120 @@
+!Feature
+next_elt_id: 4
+name: CVA6_User_Mode_Counter_CSRs(cycle, instret, cycleh, instreth)
+id: 3
+display_order: 3
+subfeatures: !!omap
+- 000_Power-on-reset (POR) values of CSR: !Subfeature
+    name: 000_Power-on-reset (POR) values of CSR
+    tag: VP_CSR_VERIFICATION_F003_S000
+    next_elt_id: 1
+    display_order: 0
+    items: !!omap
+    - '000': !VerifItem
+        name: '000'
+        tag: VP_CSR_VERIFICATION_F003_S000_I000
+        description: Upon reset, RISC-V CVA6 User mode counter CSRs must initialize
+          to their respective POR value.
+        reqt_doc: 
+          https://docs.openhwgroup.org/projects/cva6-user-manual/01_cva6_user/CV32A6_Control_Status_Registers.html
+        ref_mode: page
+        ref_page: ''
+        ref_section: ''
+        ref_viewer: firefox
+        verif_goals: Verify that the User Mode counter CSR POR value must match with
+          the value specified in the RISC-V CVA6 user manual.
+        pfc: 1
+        test_type: 1
+        cov_method: -1
+        cores: 8
+        coverage_loc: ''
+        comments: ''
+- 001_Counter _CSRs_functionality_checking: !Subfeature
+    name: 001_Counter _CSRs_functionality_checking
+    tag: VP_CSR_VERIFICATION_F003_S001
+    next_elt_id: 1
+    display_order: 1
+    items: !!omap
+    - '000': !VerifItem
+        name: '000'
+        tag: VP_CSR_VERIFICATION_F003_S001_I000
+        description: "This feature pertains to the verification of functionality of
+          RISC-V cycle, cycleh, instret and instreth Control Status Register (CSR).
+          In a RISC-V architecture\n\n1.’cycle’ and ‘cycleh’ are user-level CSR that
+          hold low 32 bits and high 32 bits respectively of the count of clock cycles
+          executed by the processor.\n2.’instret’ and ‘instreth’ are also user-level
+          CSR that count the total number of instructions executed by the processor.\n
+          \nThe functionality of user mode counter CSR is being tested by performing
+          two continuous reads and checking whether the value in the second read is
+          greater than the value in the first read."
+        reqt_doc: 
+          https://docs.openhwgroup.org/projects/cva6-user-manual/01_cva6_user/CV32A6_Control_Status_Registers.html
+        ref_mode: page
+        ref_page: ''
+        ref_section: ''
+        ref_viewer: firefox
+        verif_goals: "1.Verify that these CSR are properly initialized.\n2.Initiate
+          a second read from the counter CSR immediately after the first read.\n3.Ensure
+          that the value of the second read from counter CSR is greater than the value
+          of the initial read.\n4.Confirm that counter CSRs are correctly incrementing."
```

Comment:
We cannot say that we can "Confirm that counter CSRs are _correctly_ incrementing", only that the CSRs _are_ incrementing.

Reply:
Modified verif_goals as mentioned above i.e **Confirm that counter CSRs are incrementing.**